packages/core/src/components/dropdown/dropdown.scss

```diff
@@ -32,8 +32,10 @@
   }
 }
 
+$dropdown-offset: 3.125rem;
```

Comment:
Why exactly this value, when the ix-select e.g. has a height of 32px?
IMO we should make the offset based on the dropdown's trigger/anchor element.